lib/pages/post_page.dart

```diff
@@ -1,142 +1,108 @@
-                ),
-                Expanded(
-                  child: Container(
-                      width: double.infinity,
+    Widget buildHeader() => Row(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: [
+              Padding(
+                padding: EdgeInsets.all(5),
+                child: ProfilePicture(name: "Name", radius: 16, fontsize: 14),
+              ),
+              Padding(
+                padding: const EdgeInsets.all(5.0),
+                child: Text(item.itemName, style: theme.darkTitleStyle),
+              ),
+              const Spacer(),
+              IconButton(color: theme.dark, icon: Icon(Icons.keyboard_return_rounded), onPressed: () => routeBack(context))
+            ]);
 
-                      // move to theme
-                      decoration: theme.cardSectionDecoration,
-                      child: Padding(
-                        padding: const EdgeInsets.all(8.0),
-                        child: Column(
-                            crossAxisAlignment: CrossAxisAlignment.start,
-                            children: [
-                              RichText(
-                                  textAlign: TextAlign.start,
-                                  text: TextSpan(children: [
-                                    // Text should pull global font
-                                    TextSpan(
-                                        text: "Description\n",
-                                        style: theme.subtitleStyle),
+    Widget buildBody() => Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Text(item.description,
+              style: theme.veryDarkRegularStyle, textAlign: TextAlign.left),
+        );
 
-                                    TextSpan(
-                                        text: item.description,
-                                        style: theme.regularStyle),
-                                  ])),
-                              const Spacer(),
-                              Row(
-                                  children: generateTags() +
-                                      [
-                                        const Spacer(),
-                                        Material(
+    Widget buildFooter() => Row(children: [
+          IconButton(
+              color: theme.dark,
+              icon: Icon(Icons.map),
+              onPressed: () => routeHome(0, context)),
+          IconButton(
+              color: theme.dark,
+              icon: Icon(Icons.chat),
+              onPressed: () => routeHome(2, context)),
+          Spacer(),
+          Padding(
+            padding: const EdgeInsets.all(5.0),
+            child: Text(getDate(), style: theme.darkRegularStyle),
+          ),
+          Padding(
+              padding: const EdgeInsets.all(5.0),
+              child: Icon(Icons.access_time_sharp, color: theme.dark)),
```

Comment:
Can we swap date and clock icon order?

Reply:
I've tried both, I think it looks better how it currently is. But here's the other: 
![image](https://github.com/Steelhacks-2023/Lost-At-Pitt/assets/92108625/5efd753a-df4e-4602-bc80-697e114574c4)
Icons between posts don't line up nicely if we put it on the left